@here/harp-mapview/lib/MapView.ts

```diff
@@ -3503,15 +3503,16 @@ export class MapView extends THREE.EventDispatcher {
         // running. The initial placement of text in this render call may have changed the loading
```

Comment:
This comment should be fixed, it mentions above "first complete frame"

Reply:
done